pkg/release/release.go

```diff
@@ -103,7 +103,8 @@ const (
 	TestBucket = "kubernetes-release-gcb"
 
 	// ProductionBucket is the default bucket for Kubernetes releases.
-	ProductionBucket = "kubernetes-release"
+	// Owned by SIG k8s Infra: https://git.k8s.io/community/sig-k8s-infra
+	ProductionBucket = "767373bbdcb8270361b96548387bf2a9ad0d48758c35"
```

Comment:
where can we check this value?

Reply:
@cpanato https://github.com/kubernetes/k8s.io/blob/5a417fe27eae5891761f8f636857fcb1e1baa173/infra/gcp/terraform/k8s-infra-releases-prod/main.tf#L38

Reply:
Is it possible for us to view the contents of the bucket using a short URL?

Reply:
Yeah. I will have a PR against https://git.k8s.io/k8s.io that will allow release managers to view the content of the bucket.